website/docs/installation/configuration.mdx

```diff
@@ -150,7 +150,11 @@ These settings affect where media files are stored. Those files include applicat
 
 ### `AUTHENTIK_SECRET_KEY`
 
-Secret key used for cookie signing and unique user IDs. Do not change this after the first install.
+Secret key used for cookie signing. Changing this will invalidate active sessions.
+
+:::caution
+Prior to 2023.06.0 it was also used for unique user IDs and should not be changed after the first install.
```

Comment:
```suggestion
Prior to 2023.06.0 it was also used for unique user IDs. If running a pre-2023.06.0 version of authentik the key should not be changed after the first install.
```

Reply:
Thanks for this contribution and clarification @pidi3000! I suggested a change to the last section... if that works for you then just accept the suggestion and then we can merge it. Thanks again!

Reply:
good point, makes it more understandable.
Happy to help.